component/loki/process/stages/inspector.go

```diff
@@ -0,0 +1,133 @@
+package stages
```

Comment:
Do we want to expose this API right now? Should we move stages to an internal package?